x/claims/keeper/ibc_callbacks.go

```diff
@@ -167,6 +173,17 @@ func (k Keeper) OnRecvPacket(
 	case senderRecordFound && !recipientRecordFound && isTriggerAmt:
 		// case 2: only the sender has a claims record
 		// -> migrate the sender record to the recipient address and claim IBC action
+
+		claimedAmt := sdk.ZeroInt() // nolint: ineffassign
+
+		claimedAmt, err = k.ClaimCoinsForAction(ctx, recipient, senderClaimsRecord, types.ActionIBCTransfer, params)
+
+		// if the transfer fails or the claimable amount is 0 (eg: claims inactive or action already
+		// completed), don't perform a state migration
+		if err != nil || claimedAmt.IsZero() {
+			break
```

Comment:
@fedekunze why do you use `break` here instead of  `return ack`?

Reply:
This is just to exit the switch, the error is checked immediately after it